web_src/js/components/ContextPopup.vue

```diff
@@ -80,20 +80,25 @@ export default {
     });
   },
   methods: {
-    load(data) {
+    async load(data) {
       this.loading = true;
       this.i18nErrorMessage = null;
-      $.get(`${appSubUrl}/${data.owner}/${data.repo}/issues/${data.index}/info`).done((issue) => {
+
+      try {
+        const response = await GET(`${appSubUrl}/${data.owner}/${data.repo}/issues/${data.index}/info`);
+        if (!response.ok) throw response;
```

Comment:
It seems strange to throw a response. It's better to only throw "Error"s.

And it seems strange to use try-catch in another try-catch for the error handling. Maybe it should be:

```
try {
   if (!ok) {
      i18nErrorMessage = (await json()).message
      return
   }
} catch {
    i18nErrorMessage = i18n.network_error;
}

Reply:
Sure, done https://github.com/go-gitea/gitea/pull/29367/commits/dd978ec5ae33652efabc1a41b2f5b8511fe2e90f